.github/workflows/run-infrastructure-tests.yaml

```
@@ -38,3 +38,40 @@ jobs:
+        run: mkdir -p "$TF_PLUGIN_CACHE_DIR"
+
+      - name: Cache Terraform plugins
+        uses: actions/cache@27d5ce7f107fe9357f9df03efb73ab90386fccae  # v5.0.5
+        with:
+          key: ${{ runner.os }}-${{ runner.arch }}-terraform-${{ hashFiles('infrastructure/**/.terraform.lock.hcl') }}
+          path: ${{ env.TF_PLUGIN_CACHE_DIR }}
+
+      - name: Run integration tests
+        run: make test-infrastructure-integration
+    timeout-minutes: 10
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)